MAUI/Badge-View/Badge-customization.md

```diff
@@ -344,3 +344,30 @@ Content = sfBadgeView;
 {% endtabs %}
 
 ![BadgeView Alignment](badge-customization_images/badge_alignment.png)
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```

Comment:
The `FontAutoScalingEnabled` property is used to automatically scale the badge text's font size based on the operating system's text size. The default value of the FontAutoScalingEnabled property is false.

Reply:
Yes, changed.